app/upgrades/v11.14.0/upgrades.go

```diff
@@ -0,0 +1,55 @@
+package v11_14_0
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
+	"github.com/persistenceOne/pstake-native/v2/x/liquidstakeibc/keeper"
+	liquidstakeibctypes "github.com/persistenceOne/pstake-native/v2/x/liquidstakeibc/types"
+
+	"github.com/persistenceOne/persistenceCore/v11/app/upgrades"
+)
+
+func CreateUpgradeHandler(args upgrades.UpgradeHandlerArgs) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		ctx.Logger().Info("running module migrations...")
+
+		RemoveStargazeUnbondedBalance(ctx, args.Keepers.LiquidStakeIBCKeeper)
+
+		return args.ModuleManager.RunMigrations(ctx, args.Configurator, vm)
+	}
+}
+
+func RemoveStargazeUnbondedBalance(ctx sdk.Context, liquidStakeIBCKeeper *keeper.Keeper) {
```

Comment:
Cant we just change the state of the current unbonding so it gets processed ?

Reply:
the current unbonding will get deleted as soon as next one is processed. 
infact it is already deleted